optimum/commands/export/openvino.py

```diff
@@ -230,23 +248,17 @@ def run(self):
 
         if self.args.weight_format is None:
             ov_config = None
+            if not no_compression_parameter_provided(self.args):
+                logger.warning(
+                    "The provided compression parameters will not affect conversion because of the missing --weight-format argument."
+                )
```

Comment:
To be honest, this warning is not very visible because it gets lost with other info/warning messages related to conversion.

@AlexKoff88 What do you think about raising an exception in this case? My view is that if `--weight-format` was omitted, then no compression parameters should be accepted.